src/pipx/commands/commands.py

```diff
@@ -16,6 +15,15 @@
 from shutil import which
 from typing import List
 
+try:
+    import multiprocessing.synchronize
```

Comment:
Is it not possible just to do:
```
try:
    from multiprocessing import Pool
except ImportError:
    from multiprocessing.dummy import Pool
```
```

Reply:
Nope. Per the stack trace in #229, the error occurs when a Pool is constructed, not imported, at which point `multiprocessing.synchronize` fails to import the needed semaphore classes from `_multiprocessing`.

```
File "/data/data/com.termux/files/home/.local/pipx/venvs/pipx/lib/python3.7/site-packages/pipx/commands.py", line 638, in list_packages
    with multiprocessing.Pool() as p:
  File "/data/data/com.termux/files/usr/lib/python3.7/multiprocessing/context.py", line 119, in Pool
  # snipped...
    File "/data/data/com.termux/files/usr/lib/python3.7/multiprocessing/context.py", line 66, in Lock
    from .synchronize import Lock
```

I see that flake8 complains about the unused import, that's my bad. I ran `nox` locally but didn't see any warnings during linting, will investigate on my end.

I can import both a `Pool` and a `ThreadPool` and move the `try` down to when the Pool is instantiated if that's preferable to a `#noqa`.

Or just switch to threads, your call.

Reply:
I'd lean toward `# noqa` with an explanatory comment for why we are trying to import something we are not using (a brief version of your above comment.)

Probably @cs01 should chime in on hist preferences.